rtl/ibex_core.f

```diff
@@ -12,5 +12,5 @@
 ibex_multdiv_fast.sv
 ibex_prefetch_buffer.sv
 ibex_fetch_fifo.sv
-ibex_register_file_ff.sv
+ibex_register_file_fpga.sv
```

Comment:
Same comment: I think we should keep using the FF-based Regfile by default.

Reply:
Done. Thank you!